eos-updater/com.endlessm.Updater.xml

```diff
@@ -2,49 +2,297 @@
 '-//freedesktop//DTD D-BUS Object Introspection 1.0//EN'
 'http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd'>
 <node>
+  <!--
+    com.endlessm.Updater:
+    @short_description: Endless OS updater control interface
+
+    Interface to query the state of the OS updater, and to control it: checking
+    for, downloading and applying updates.
+
+    The updater moves through a series of states (see the `State` property and
+    `StateChanged` signal), with state transitions triggered by the appropriate
+    method call, or by an error in an operation. It will return a
+    `com.endlessm.Updater.Error.WrongState` error if you attempt to perform an
+    invalid state transition.
+  -->
   <interface name="com.endlessm.Updater">
+    <!--
+      Poll:
+
+      Check for updates. This may be performed from the `Ready`,
+      `UpdateAvailable`, `UpdateReady` or `Error` states. It will immediately
+      transition to the `Polling` state. If an update is then successfully
+      found, it will transition to the `UpdateAvailable` state. If no update is
+      available, it will transition to the `Ready` state. If there is an error
+      checking for an update, it will transition to the `Error` state.
+
+      If an update is found, its details will be reported through the updater’s
+      properties once the updater reaches the `UpdateAvailable` state. In
+      particular, through the `UpdateID`, `UpdateRefspec`, `UpdateLabel` and
+      `UpdateMessage` properties.
+    -->
     <method name="Poll"></method>
+
+    <!--
+      PollVolume:
+      @path: Absolute path to the root directory of a volume to check.
+
+      Like `Poll`, except this polls a specific removable volume (such as a USB
+      stick), rather than the internet.
+
+      If a `.ostree/repo` directory is available beneath @path, and it contains
+      a valid OSTree repository, that repository will be checked for updates.
```

Comment:
And if not? Does the state change to Error?

Reply:
It’ll go to `Ready`, afaik, since no update is available.